src/lib/components/quiz/more-info.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+  export let title = '';
+  export let picture = '';
+  export let color = '';
+  export let blurp = '';
+  export let blurpRecommend = '';
+  export let recommendations: Array<{ title: string; link: string }> = [];
+  export let workshopPictures: string[] = [];
+</script>
+
+<section class="container" style={`--team-color: ${color}`}>
```

Comment:
I JUST remembered the Svelte shorthand I was thinking of when originally reviewing this code.
```html
<section class="container" style:--team-color={color}>
```